src/components/WebexAvatar/WebexAvatar.jsx

```diff
@@ -24,18 +34,51 @@ export default function WebexAvatar({
 }) {
   const {avatar, displayName, status} = usePerson(personID);
   const type = displayStatus ? status : null;
-
+  const iconName = status;
```

Comment:
do icons really have the same name as statuses, or did you name them as such? I would prefer to keep the figma icon names if possible, and create a map here, eg `statusIcons`.

the status class names, on the other hand, can be easily computed instead of using a map:
```js
statusClassName = 'avatar-status-' + status;
```